### PR TITLE
Add toolchain dependencies to linux build_tests.

### PR DESCRIPTION
**Pull request by godofredoc**
This is support build_tests for linux, the dependencies were previously
added only to prod builders.

### CODE REVIEW

**Review by keyonghan — APPROVED** [review]
LGTM

File: config/lib/common.star [hunk 1]
```diff
@@ -30,7 +30,8 @@ def _repo_url_to_luci_object_name(repo_url):
     prefix = domain
     if domain.endswith(".googlesource.com"):
         prefix = domain[:-len(".googlesource.com")] + "-"
-    return prefix + "-".join(path).replace(".", "_")
+
+    return "-".join(path).replace(".", "_")
```
Comment (stuartmorgan) [inline config/lib/common.star:34]: `prefix` is still being computed, but is now unused.

Comment (godofredoc) [reply config/lib/common.star:28]: Removed those lines.

**Review by stuartmorgan — APPROVED** [review]
LGTM

**Pull request merged**